lib/page/setting.dart

```diff
@@ -0,0 +1,248 @@
+// ignore_for_file: prefer_const_constructors
+
+import 'package:flutter/material.dart';
+import 'package:meta/meta.dart';
+import 'package:flutter/rendering.dart';
+import 'package:homework_flutter/page/homepage.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:go_router/go_router.dart';
+import 'package:homework_flutter/main.dart';
+
+/// The screen of the second page.
+class Page2Screen extends StatefulWidget {
```

Comment:
Should name class same as file name and if it is a page should ended with 'page' e.g. SettingPage